content/docs/traffic-management/_index.md

```diff
@@ -19,4 +19,5 @@ weight: 400
   {{< card link="session-affinity" title="Session affinity" >}}
   {{< card link="transformations" title="Transformations" >}}
   {{< card link="traffic-split" title="Traffic splitting" >}}
+  {{< card link="weighted-routes" title="Route weighting" >}}
```

Comment:
maybe `Weight-based routing`? or `Weighted routing`?

Reply:
I used `Weighted routing` since it's shorter, and we also used the phrase `weight-based routing` for the traffic splitting page 😅